fuzzers/007-timing/bel/tim2json.py

```diff
@@ -63,26 +93,59 @@ def pin_in_model(pin, model, direction=None):
             return True, pin
         elif extended_pin_name in model.split('_'):
             return True, extended_pin_name
+        elif (pin_aliases is not None) and (pin.upper() in pin_aliases):
+            for alias in pin_aliases[pin.upper()]['names']:
+                pin_alias = alias.lower()
+                if pin_alias in model.split('_'):
+                    if pin_aliases[pin.upper()]['property']:
+                        return True, pin
+                    else:
+                        return True, pin_alias
+            return False, None
         else:
             return False, None
+        return False, None
     else:
         # pin name is multi word, search for a string
-        return (pin in model), pin
+        if pin in model:
+            return True, pin
+        elif pin_aliases is not None:
+            for alias in pin_aliases.get(pin.upper(), ()):
+                pin_alias = alias.lower()
+                if pin_alias in model:
+                    return True, pin
```

Comment:
This loop can fall off without a return?  Missing assert?

Reply:
the assert is not necessary after refactoring